cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -115,6 +116,10 @@ def filtered_list(cls):
         if not switches.is_enabled(u'instructor_paced_only'):
             filtered_list.append('certificate_available_date')
 
+        # Adaptive Learning should be turned off by default unless enabled.
```

Comment:
@macornwell Thanks for adding this comment! Let's be even more precise, though: The logic below doesn't turn off Adaptive Learning completely, it just hides the settings from "Advanced Settings". So how about:

> Do not show field for configuring Adaptive Learning if feature is disabled.

(This format also matches the format of the remaining comments in `filtered_list`.)